actors/evm/src/types.rs

```diff
@@ -0,0 +1,11 @@
+use fil_actors_evm_shared::address::EthAddress;
```

Comment:
Drop this?

Reply:
Oh, right, I wanted to move this stuff into a `types` file. I'll drop it from the lib source file.